docs/participate/treasury-proposal.md

```diff
@@ -0,0 +1,53 @@
+---
+id: treasury-proposal
+title: Create a treasury proposal
+---
+
+Complete these steps to create a well-formed Treasury proposal
+
+# Discuss
+
+The first step in applying for a Treasury grant is to join the community in the [Element Direction Channel](https://matrix.to/#/#kilt-direction:matrix.org), [Discord Treasury Channel](https://discord.gg/nUpqDfQ6kJ) or on [Telegram](https://t.me/KILTProtocolChat) to discuss and share your intended proposal. This will help you get valuable community feedback throughout the process. This also gives Council members an open and transparent way to measure community sentiment.  
+
+# Deposit
+
+A deposit of 5% of the amount requested is required in order to submit a proposal. If the proposal is denied, you will lose this amount and it will go to the Treasury to fund other projects. This is why it is essential to engage with the community and show how the proposed work adds value to the network. 
+
+# Proposal Document
+
+To maximize your chances of success, create a full proposal document with as much information as possible to communicate the value of your work and what it will add to the network’s growth and success. Check out our Treasury proposal template or Polkassembly’s [proposal document #6](https://docs.google.com/document/d/1NilDtFljCc6boyHkz2o2kGyFNhR6hVQSGzz7J0Xhwco/edit) as examples to help guide your process.
+
+Once your proposal document is complete, upload it so that it’s accessible to the Council for review. Don’t forget to link it when completing the Polkassembly information! 
```

Comment:
New line

```suggestion
Once your proposal document is complete, upload it so that it’s accessible to the Council for review.
Don’t forget to link it when completing the Polkassembly information! 
```